BootTidal.hs

```diff
@@ -6,7 +6,7 @@ import Sound.Tidal.Context
 import System.IO (hSetEncoding, stdout, utf8)
 hSetEncoding stdout utf8
 
-tidal <- startTidal (superdirtTarget {oLatency = 0.1, oAddress = "127.0.0.1", oPort = 57120}) (defaultConfig {cVerbose = True, cFrameTimespan = 1/20})
+tidal <- startTidal (superdirtTarget {oLatency = 0.05, oAddress = "127.0.0.1", oPort = 57120}) (defaultConfig {cVerbose = True, cFrameTimespan = 1/20})
```

Comment:
Just to be sure, is your comment based on before or after the change to use Ableton Link for scheduling?

To avoid late messages, cProcessAhead can be adjusted. It's current value `3/10` instructs Tidal to schedule events when they are 300ms away on the Link timeline.

oLatency adjusts how much time SuperDirt needs for sound synthesis. A higher value will start the synthesis earlier. A higher oLatency requires a higher value for cProcessAhead if we want to avoid late messages.

It would be great to have a report from different people to learn what settings they use. That input could help us set better defaults. Since scheduling has changed significantly, I did not trust the old values and thus changed them to fit my setup. My thinking was that it's better to have values that have been tested with the new scheduling than to stick to the old values.

Reply:
I always thought (and it always behave in the way...) that the `oLatency` value is added to the timestamp that's sent to superdirt to trigger the sounds, and when I set it under 0.2 late messages start to appear.

Reply:
You are correct about the old behaviour @ndr-brt! I hope to make a technical writeup about the new behaviour in early fall as part of the documentation that needs to be added. cProcessAhead achieves the same effect but in a different way, while oLatency is now only useful for alignment on the Link timeline.